docs/htdocs/info/docs/tools/vep/script/vep_plugins.html

```diff
@@ -38,17 +38,19 @@
   <h1 id="top"><span style="color:#006;padding-right:15px">Variant Effect Predictor</span><span style="color:#666"><img src="/i/16/tool.png"/> Plugins</span></h1>
   <hr/>
         
-  <p> VEP can use plugin modules written in Perl to <b>add functionality</b> to
-  the software.</p>
-  <p>Plugins are a powerful way to <b>extend</b>, <b>filter</b> and <b>manipulate</b> the
-  VEP output.<br />They can be installed using VEP's installer script, run the
-  following command to get a list of available plugins: </p>
+  <p> VEP can use plugin modules written in Perl to <b>extend, filter and manipulate</b> the VEP output. </p>
 
-  <pre class="code sh_sh">perl INSTALL.pl -a p -g list</pre>
+  <p> To use plugins with VEP, you can: </p>
+  <ul>
+    <li> Install them using <a href="vep_download.html#installer">VEP's installer script</a>. You can quickly check installed plugins by running:<li>
+    <ul>
+      <li> <pre class="code sh_sh">perl INSTALL.pl -a p -g list</pre> </li>
+    </ul>
```

Comment:
```suggestion
    <li> Install them using <a href="vep_download.html#installer">VEP's installer script</a>. You can quickly check installed plugins by running:
    <ul>
      <li> <pre class="code sh_sh">perl INSTALL.pl -a p -g list</pre> </li>
    </ul>
    </li>
```

Currently the nested list is not working.

Reply:
Fixed, thanks @nakib103!